packages/sirius-web/backend/sirius-web/src/main/resources/application.properties

```diff
@@ -29,4 +29,8 @@ sirius.components.cors.allowedOriginPatterns=*
 sirius.components.cors.allowedCredentials=true
 
 sirius.web.graphql.tracing=true
-logging.group.sirius.web.performance=org.eclipse.sirius.web.application.editingcontext.services.EditingContextSearchService,org.eclipse.sirius.web.application.editingcontext.services.EditingContextPersistenceService,org.eclipse.sirius.components.collaborative.editingcontext.EditingContextEventProcessor,org.eclipse.sirius.components.interpreter.AQLInterpreter
\ No newline at end of file
+logging.group.sirius.web.performance=org.eclipse.sirius.web.application.editingcontext.services.EditingContextSearchService,org.eclipse.sirius.web.application.editingcontext.services.EditingContextPersistenceService,org.eclipse.sirius.components.collaborative.editingcontext.EditingContextEventProcessor,org.eclipse.sirius.components.interpreter.AQLInterpreter
+
+spring.elasticsearch.uris=http://localhost:9200
+spring.elasticsearch.username=elastic
+spring.elasticsearch.password=<YOUR PASSWORD>
```

Comment:
You should update this with the password you generated when installing Elasticsearch